custom_container/custom_container_iscsi.yaml

```diff
@@ -0,0 +1,61 @@
+parameter_defaults:
+  DockerCinderVolumeImage: <undercloud_full_hostname>:8787/rhosp-rhel8/openstack-cinder-volume-hpe:1.0
```

Comment:
Drop 1.0 tag and use latest instead. Refer https://github.com/hpe-storage/hpe-3par-cinder-rhosp13/blob/master/custom_container/custom_container_iscsi.yaml#L2.

Reply:
Ok, Will update here also.